templates/footer.html

```diff
@@ -0,0 +1,11 @@
+<footer class="page-footer">
+    <div class="footer-copyright">
+        <div class="container">
+            <span class="left">© 2017 {{ org_name }}</span>
```

Comment:
`2017` should be a generated value.

but more importantly, this implies something different than https://github.com/coala/community/blob/master/LICENSE says